src/validations/basic/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Validation",
+  "definitions": {
+    "Validation": {
+      "title": "Basic validation",
+      "type": "object",
+      "properties": {
+        "minScore": {
+          "title": "Minimum score",
+          "type": "number",
+          "minimum": 1
+        },
+        "useVotingStrategies": {
+          "description": "Use voting strategies",
+          "type": "boolean",
+          "default": false
+        },
+        "strategies": {
+          "title": "Strategies",
+          "type": "array",
+          "items": {
+            "title": "Strategy",
+            "type": "object",
+            "default": {
+              "name": "erc20-balance-of",
+              "network": "1",
+              "params": {}
+            },
```

Comment:
I think instead of erc20 better to use the `ticket` strategy as the default

Reply:
Done